src/utils/money.cljs

```diff
@@ -47,11 +47,10 @@
   (if (bignumber? n) n (bignumber n)))
 
 (defn ->bignumbers
-  [n1 n2]
-  (when-let [bn1 (->bignumber n1)]
-    (when-let [bn2 (->bignumber n2)]
-      (when (and (bignumber? bn1) (bignumber? bn2))
-        [bn1 bn2]))))
+  [& numbers]
+  (let [transformed-numbers (map ->bignumber numbers)]
+    (when (every? bignumber? transformed-numbers)
+      transformed-numbers)))
```

Comment:
Made this function more general, so it works as before but now we can pass any number of numbers, before only accepted 2